### DISCUSSION TITLE
Support for ARM-based Linux systems (aarch64)

### DISCUSSION THREAD

**Discussion by rwiesmayr** [mitsuba-renderer/mitsuba3#1433] [Feature requests] [answered]
Dear Mitsuba Developers,

I’m trying to use Mitsuba 3.6.0 on an ARM-based Linux system (NVIDIA GH200). I noticed that official support is currently available for x86-64 Windows and Linux, as well as macOS x86-64 and ARM64, but not for aarch64 Linux systems.

Are there any plans to provide official support for aarch64 Linux in the near future?

Thank you in advance for your help.

Best regards,
Reinhard Wiesmayr

**Comment by merlinND** [comment]
Adding a note for Windows ARM support as well, which was mentioned in https://github.com/mitsuba-renderer/mitsuba3/issues/1437.

**Comment by njroussel** [comment, marked as answer]
There's currently no plan to support these platforms (ARM Linux & Windows).

I can't think of a big blocker that would make it difficult to run on these targets, I only expect some straightforward changes in the codebase. However we don't really have the means, needs, nor tools to officially support this at the moment, hence why it hasn't been done.
